public/js/controllers/mempool.js

```diff
@@ -232,5 +184,42 @@
                 addTxRow(tx, this[tx.Type.toLowerCase() + "TransactionsTarget"], rowFn)
             })
         }
+
+        filterVotesLastBlock() {
```

Comment:
One last nit about code formatting. 
I'm not into bike-shedding about semi-colons or no semi-colons, and don't think we need a js linter to enforce style. But I do think we it is beneficial to have a baseline of standards.

I think this is the smallest set of rules that are worth aiming for as a team for the sake of having a relatively consistent js codebase. https://crockford.com/javascript/code.html 

So in terms of this function, there should be whitespace between control flow characters, comparators, and variable assignments.
eg `var bestBlockHash = $(this.bestBlockTarget).data("hash");`
and `if (voteBlockHeight > bestBlockHeight) {`

Reply:
I actually was curious what my protocol should be on semi-colons.  I just pushed those changes.  I'll try to assimilate the code.html strategies you sent over as I go.  I appreciate the feedback - the more aligned we all are to a common theme the more professional we all look.

Reply:
I tend to leave them out.
If I had to chose a linter it would probably be https://standardjs.com/ which is the main one people that who don't use semi-colons use. I point people to the crockford site since that's like 20 years old and still pretty much looks like most of the well written js you see out there today, so I figure that's evidence that the recommendations are reasonable.  Not even TC39 can decide on the semi-colons though lol https://github.com/tc39/agendas/commit/c0b54452ff2e2366e6e7847559ef237d5b2f6783#diff-5a8c1d412a7069f29901ea548d76ac26